src/clj_bugsnag/core.clj

```diff
@@ -54,6 +54,9 @@
     thing
     (str thing)))
 
+(defn- environment [options]
+  (or (:environment options) "production"))
```

Comment:
this was like this before, but better code style would be to use 
`(:key map "defualt")`